config/samples/tlspolicy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kuadrant.io/v1alpha1
+kind: TLSPolicy
+metadata:
+  name: example-tlspolicy
+spec:
+  targetRef:
+    name: prod-web
+    group: ""
+    kind: Gateway
+  issuerRef:
+    group: cert-manager.io
+    kind: ClusterIssuer
+    name: glbc-ca
+  secretName: todo-host
```

Comment:
hmm how does this work with multiple listeners?

Reply:
so this can be used independent of DNS as the issuer will have the credentials, nice